src/Symfony/Component/Console/Command/Command.php

```diff
@@ -447,6 +448,30 @@ public function getName()
     }
 
     /**
+     * To hide the command in application descriptions.
+     *
+     * @param bool $public To show this command or not
```

Comment:
I'd suggest something like: `Whether the command should be publicly shown or not.` and remove the above description (which brings nothing more IMHO).